main/src/kotlinx/team/infra/Utils.kt

```diff
@@ -1,13 +1,12 @@
 package kotlinx.team.infra
 
-import groovy.lang.*
 import org.gradle.api.*
 import org.gradle.api.logging.*
 import org.gradle.api.model.ObjectFactory
 import org.gradle.api.tasks.*
 import java.io.*
 
-fun cleanup(file: File) {
+internal fun cleanup(file: File) {
```

Comment:
This function seems to be redundant now. It was previously used by ApiCheck tasks.

Reply:
Good eye!